### PR TITLE
Update orientation material

### PR DESCRIPTION
**Pull request by don-smith**

### CODE REVIEW

File: contact-and-support/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 
-1. **Any Phase 0 questions, concerns or feedback**: <mailto:rupert@devacademy.co.nz>, or contact your facilitator on Slack.
-2. **Billing, tuition, payments, refund policies etc**: <mailto:accounts@devacademy.co.nz>.
-3. **To change your cohort or withdraw**: <mailto:rupert@devacademy.co.nz>.
-4. **Technical support**: Reach out on Slack to your Phase 0 support people.
+1. **For general questions or feedback**: <mailto:jack@devacademy.co.nz>,
+2. **For questions about course material or technical support** contact your facilitator on Slack.
+3. **For billing, tuition, payments, refund policies **: <mailto:accounts@devacademy.co.nz>.
+4. **To change your cohort or withdraw**: <mailto:jack@devacademy.co.nz>.
+5. **For complaints:** <mailto:sarrah@devacademy.co.nz>
```
Comment (don-smith) [inline contact-and-support/README.md]: Move colon: `complaints**: <mailto`

File: installation/README.md [hunk 2]
```diff
@@ -1,204 +1,121 @@
...
+<br>
 
 
 ## OS X (Mac) instructions
 
-### Terminal
+### iTerm and Homebrew
 
-We recommend you use the [iTerm2](https://www.iterm2.com/) terminal. Although you _could_ complete Phase 0 without it, it's extremely popular among OS X developers, has a lot of extra features and makes things look nicer in the default configuration. Download and install it from the linked site. There's a terminal called, oddly enough, 'terminal' provided with OSX, but iTerm2 is better.
+1. Navigate to [iTerm2](https://www.iterm2.com/)
```
Comment (don-smith) [inline installation/README.md]: We really can leave out the iTerm installation. The installation isn't hard, but Mac OS already has a capabile terminal installed by default. If we're trying to remove all the noise, I think that includes iTerm.

File: accounts/README.md [hunk 3]
```diff
@@ -1,45 +1,73 @@
 # Create Accounts!
 
+A couple of things to help us along:
 
-Ngā mihi nui kia koutou katoa! Warm greetings to you all! Welcome to Dev Academy.
-
-You will need to set up these accounts before starting Phase 0. The general rule of thumb is to **use your first name as part of your username wherever possible** and include a photo of yourself (i.e. not your cat or bike). This helps us (the EDA community of staff, teachers, coaches and alumni) to better support you.
+1. **Use your first name as part of your username wherever possible.** (To help us all recognise each other)
+2. **Open links in new tabs.** GitHub repos have a lot of links. We recommend right-clicking (or holding the command key (mac) / control key (win) when clicking) to open links in new tabs.
+3. **Skim read TLDR sections.** We use TLDR to indicate the information is useful but not essential for completing the task right now.
```
Comment (don-smith) [inline accounts/README.md]: TLDR stands for "Too long don't read", which is intended to be a summary of a much longer read. I don't think we're using this initialism the correct way in these documents. Perhaps we should just title these sections as "Context"?

File: installation/README.md [hunk 4]
```diff
@@ -1,204 +1,121 @@
...
+## Installs
+
+1. iTerm2 (Mac) or Babun(Windows)
+2. Homebrew (Mac only)
+3. Git
+4. VS code
+
+<br>
+---------------------------------------
+<br>
```
Comment (don-smith) [inline installation/README.md]: Remove

File: student-handbook.md [hunk 5]
```diff
@@ -1,8 +1,25 @@
...
+6. [Best practices for success](/best-practices/)
+7. [Changing cohorts or withdrawing](/changing-cohorts-or-withdrawing)
+8. [Contact and support](/contact-and-support)
+10. [Research and debugging](/research-and-debugging)
+11. [Use of space](/use-of-space)
+12. [Engineering Empathy](/empathy/)
+
+
+<br>-------------------------
+<br>
```
Comment (don-smith) [inline student-handbook.md]: Remove

File: student-handbook.md [hunk 6]
```diff
@@ -11,21 +28,4 @@ We recommend you be familiar with these tools and technologies:
 3. [Web accessibility](/accessibility/) (15 minutes)
```
Comment (don-smith) [inline student-handbook.md]: I thought we were removeing all of the time estimations.

**Review by don-smith — COMMENTED** [review]
Need to add `.DS_Store` to the `.gitignore` file and delete the file.

File: README.md [hunk 7]
```diff
@@ -1,14 +1,15 @@
...
-1. [Sign up for accounts](/accounts) (10 minutes)
-2. [Prepare for introduction session](prepare-for-orientation) (10 minutes)
-3. [Complete the preparation form](https://docs.google.com/forms/d/1-MW9w5sHtyWZCoFFyDoIqFU8xgDClGIQug2ufACy0-4/viewform) (2 minutes)
-
-Once you're finished the above steps, sit back, relax, and enjoy your Orientation
-
-Afterwards, there is [further preparation material to read](course-prep.md) for more information about the course, our expectations of you as a student, and how to get setup to start developing.
+## Post-Orientation
+1. [Read the Student handbook](student-handbook.md)
+2. [Complete Personal Information form]()
```
[the remark this answers is not in the document]
Comment (Jamanius) [reply README.md]: Happy to use the same link - the form content will need to be updated however.

File: accounts/README.md (hunk 3 above)
Comment (Jamanius) [reply accounts/README.md]: Context is good. Thanks @don-smith

File: README.md [hunk 8]
```diff
@@ -1,14 +1,15 @@
-# Get ready for the course!
+# Ngā mihi nui kia koutou!
+### Welcome to orientation!
```
[the remark this answers is not in the document]
Comment (Jamanius) [reply README.md]: @don-smith - They were indeed meant to be headers! I've updated "Welcome to orientation" as a bold instead.

File: README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Jamanius) [reply README.md]: New form is linked

File: best-practices/README.md [hunk 9]
```diff
@@ -1,12 +1,12 @@
 # Best practices for success
 
-The **Phase 0** material is organized into a series of nine "sprints", each approximately 25hrs in length for the typical student. You can move faster or slower, but after the first couple of sprints you'll have a good idea of your pace.
+The **Phase 0** material is organized into a series of nine "sprints", each approximately 25hrs in length for the typical student.You can move faster or slower, but after the first couple of sprints you'll have a good idea of your pace.
```
[the remark this answers is not in the document]
Comment (Jamanius) [reply best-practices/README.md]: @don-smith I'm struggling here. What is the change you'd like me to undo?

File: best-practices/README.md [hunk 10]
```diff
@@ -23,9 +23,9 @@ We'll give you time estimates for these parts. These will range widely as some o
 
 ## Skim the material
 
-Before you get started, skim through the material to see what's involved and to get a feel for how to divide your time and effort. Sometimes material is sequential&mdash;you need to complete one item before you can move on to the next&mdash;but often you can do things out of order. So keep that in mind.
+Before you get started, skim through the material to see what's involved and to get a feel for how to divide your time and effort.Sometimes material is sequential&mdash;you need to complete one item before you can move on to the next&mdash;but often you can do things out of order. So keep that in mind.
```
[the remark this answers is not in the document]
Comment (Jamanius) [reply best-practices/README.md]: @don-smith : I'm not sure what the change is here?

File: changing-cohorts-or-withdrawing/README.md [hunk 11]
```diff
@@ -17,14 +22,16 @@ If repeating is an option you want to explore more, please talk to your facilita
...
 
 If you have decided to leave the program, there are a few steps we ask you to take to do this.
 
-**If you have not started Phase 0** yet, or are in the Week 0 module, please contact both <mailto:admissions@devacademy.co.nz> and your Phase 0 facilitator to let us know of your decision.
+**If you have not started Phase 0** yet, or are in the Week 0 module, please contact both
+<mailto:admissions@devacademy.co.nz> and your Phase 0 facilitator to let us know of your decision.
 
 You will need to complete the [withdrawal form](http://goo.gl/forms/heEV7aH0zh).
 
+
```
[the remark this answers is not in the document]
Comment (Jamanius) [reply changing-cohorts-or-withdrawing/README.md]: The withdrawl form change?

File: student-handbook.md (hunk 6 above)
Comment (Jamanius) [reply student-handbook.md]: We are: I hadn't got to this section yet.

File: best-practices/README.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (don-smith) [reply best-practices/README.md]: @Jamanius it looks like a space was removed just before the second sentence that starts with `You can move ...`

File: best-practices/README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (don-smith) [reply best-practices/README.md]: @Jamanius same thing here. A space was removed just before the second sentence.

File: best-practices/README.md [hunk 12]
```diff
@@ -23,9 +23,9 @@ We'll give you time estimates for these parts. These will range widely as some o
...
 
-## Create a plan of attack (a strategy)
+## Create a plan / strategy
```
[the remark this answers is not in the document]
Comment (don-smith) [reply best-practices/README.md]: @Jamanius looks like you missed this one.

File: changing-cohorts-or-withdrawing/README.md [hunk 13]
```diff
@@ -17,14 +22,16 @@ If repeating is an option you want to explore more, please talk to your facilita
 
 ## Deferring to a Later Cohort
```
[the remark this answers is not in the document]
Comment (don-smith) [reply changing-cohorts-or-withdrawing/README.md]: @Jamanius missed this one too.

**Pull request merged**